mycore-pi/src/main/java/org/mycore/pi/cli/MCRPICommands.java

```diff
@@ -202,4 +203,38 @@ public static void updateObject(String objectIDString) {
         MCRPersistentIdentifierEventHandler.updateObject(object);
     }
 
+    @MCRCommand(syntax = "create pi with {0} for object {1} with additional ({2})",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 137).](https://app.codacy.com/app/MyCoRe/mycore/pullRequest?prid=4254170)